domains/client/domain-operator/src/snap_sync.rs

```diff
@@ -5,7 +5,6 @@ use sc_client_api::{AuxStore, Backend, ProofProvider};
 use sc_consensus::{
     BlockImport, BlockImportParams, ForkChoiceStrategy, ImportedState, StateAction, StorageChanges,
 };
-use sc_consensus_subspace::archiver::FINALIZATION_DEPTH_IN_SEGMENTS;
```

Comment:
Please make this constant non-public again

Reply:
Ok. I'd consider moving the constant next time we make it public.